website/source/docs/agent/cloud-auto-join.html.md

```diff
@@ -114,7 +114,7 @@ Use these configuration parameters when using tags:
 - `tag_name` - the name of the tag to auto-join on.
 - `tag_value` - the value of the tag to auto-join on.
 
-Use these configuration parameters when using Virtual Machine Scale Sets (Consul 1.0.3 and later):
+Use these configuration parameters (and not `tag_name` and `tag_value`) when using Virtual Machine Scale Sets (Consul 1.0.3 and later):
```

Comment:
```suggestion
Use these configuration parameters (instead of `tag_name` and `tag_value`) when using Virtual Machine Scale Sets (Consul 1.0.3 and later):
```

Reply:
I like it! I'll get it done by EOD!